karma.conf.ci.js

```diff
@@ -33,23 +33,6 @@ var customLaunchers = {
     browserName: 'safari',
     version: '9.0'
   },
-  // FIXME(ndhoule): Bad IE7/8 support in testing packages make these fail
-  // sl_ie_7: {
-  //   base: 'SauceLabs',
-  //   browserName: 'internet explorer',
-  //   version: '7'
-  // },
-  // sl_ie_8: {
-  //   base: 'SauceLabs',
-  //   browserName: 'internet explorer',
-  //   version: '8'
-  // },
-  // Satismeter doesn't work in IE9
```

Comment:
@jsedlacek I'd like to leave these in for convention purposes and just do inline comments since future engineers might wonder why we are omitting these tests. If you officially do not support below IE9, we can disable these tests and leave a note.